demo/predictions/js/data.js

```diff
@@ -0,0 +1,128 @@
+class DataExecutionManager {
+
+    _customerEvtBasedGwId = '_6-180';
+
+    _vendorBakeThePizzaId = '_6-463';
+
+    _vendorWhereIsMyPizzaId = '_6-674';
+
+    _runningElementsWithoutPrediction = [this._vendorWhereIsMyPizzaId, this._customerEvtBasedGwId];
+
+    _commonExecutedElements = [
+        // customer
+        '_6-61', // start event
+        '_6-125', // sequence flow between 'start event' and 'select pizza'
+        '_6-74', // select a pizza
+        '_6-178', // sequence flow between 'select a pizza' and 'order a pizza'
+        '_6-127', // order a pizza
+        '_6-420', // sequence flow between 'order a pizza' and 'event-based gateway' (running element)
+
+        '_6-638', // message flow
+
+        // vendor
+        '_6-450', // order received
+        '_6-630', // sequence flow between 'order received' and 'parallel gateway'
+        '_6-652', // parallel gateway
+        '_6-693', // sequence flow between 'parallel gateway' and 'Bake the pizza'
+        '_6-691', // sequence flow between 'parallel gateway' and 'where is my pizza'
+    ];
+
+    get executedElements() {
+        return [];
+    }
+
+    get runningElementWithPrediction() {
+        return null;
+    }
+
+    get predictedPaths() {
+        return null;
+    }
+
+    get runningElementsWithoutPrediction() {
+        return this._runningElementsWithoutPrediction;
+    }
+}
+
+class PredicatedLateDataExecutionManager extends DataExecutionManager {
+
+    _runningElementWithPrediction = {
```

Comment:
**thought:** could be made private. This applies to all properties only used in the class and to the `PredictedOnTimeDataExecutionManager` class